.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - 3.5
```

Comment:
I think we should support 3.5 for now, since databroker does. Any particular reason not to? We should be consistent about this, anyway.

Reply:
I see. Yes, we don't build packages for it because we don't deploy 3.5 at NSLS-II, but our packages still support 3.5. I believe some of our collaborators still rely on that.